routers/install/install.go

```diff
@@ -148,8 +148,8 @@ func Install(ctx *context.Context) {
 
 	// Server and other services settings
 	form.OfflineMode = setting.OfflineMode
-	form.DisableGravatar = false       // when installing, there is no database connection so that given a default value
-	form.EnableFederatedAvatar = false // when installing, there is no database connection so that given a default value
+	form.DisableGravatar = setting.DisableGravatar             // when installing, there is no database connection so that given a default value
+	form.EnableFederatedAvatar = setting.EnableFederatedAvatar // when installing, there is no database connection so that given a default value
```

Comment:
New patch uses setting like before, which should calculate the same as it did before.